### ISSUE TITLE
`npm run build` shows a warning – missing default export `getAppearanceType` imported by `./client/checkout/woopay/express-button/index.js`

### ISSUE THREAD

**Issue by Jinksi** [Automattic/woocommerce-payments#9168]
### Describe the bug

The following warning appears when running `npm run build` or `npm run hmr`, due to `./client/checkout/woopay/express-button/index.js` importing the default export from `wcpay/checkout/utils` rather than the named export `getAppearanceType`.

```
WARNING in ./client/checkout/woopay/express-button/express-checkout-iframe.js 85:27-44
export 'default' (imported as 'getAppearanceType') was not found in 'wcpay/checkout/utils' (possible exports: getAppearanceType, normalizeCurrencyToMinorUnit)
 @ ./client/checkout/woopay/express-button/woopay-express-checkout-button.js 13:0-66 121:8-29 124:6-27
 @ ./client/checkout/woopay/express-button/index.js 11:0-79 33:39-66
```

The fix _may_ be a simple one – import the exported named function [in `./client/checkout/woopay/express-button/index.js`](https://github.com/Automattic/woocommerce-payments/blob/efd5addea2d204c3ebf309503cc977b7078a5b5c/client/checkout/woopay/express-button/express-checkout-iframe.js#L20):
```diff
- import getAppearanceType from 'wcpay/checkout/utils';
+ import { getAppearanceType } from 'wcpay/checkout/utils';
```

### To Reproduce

1. Check out `develop`
2. `npm run build` or `npm run hmr`
3. See error above in CLI output

### Additional context

Related to PR #9138 cc @hsingyuc

**Comment by pierorocca** [comment]
@bborman22 for extra awareness.

**Comment by bborman22** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Fractal-62a0f7b955648b001e1f5e1f/issues/Automattic/woocommerce-payments/9168?planning-poker) with Zenhub @asumaran @cesarcosta99 @lovo-h @rafaelzaleski @reykjalin @ricardo

**Comment by cesarcosta99** [comment]
This has been fixed by #9105. That warning is no longer displayed when running `npm run build` or `npm run hmr`.
